sable_history/src/server/update_handler.rs

```diff
@@ -131,7 +135,11 @@ impl HistoryServer {
         }
     }
 
-    async fn handle_new_message(&self, new_message: update::NewMessage) -> anyhow::Result<()> {
+    async fn handle_new_message(
+        &self,
+        new_message: update::NewMessage,
+        update_timestamp: i64,
```

Comment:
Why is this not using the ts field from the Message?

Reply:
To make the `@time` tag consistent between echo-message and chathistory https://github.com/Libera-Chat/sable/pull/143/commits/d56db83b2b385dc211a590685b3e1263016f9a91